src/components/UserInfo.js

```diff
@@ -1,19 +1,26 @@
 export default class UserInfo {
-  constructor(nameSelector, jobSelector) {
+  constructor(nameSelector, jobSelector, avatarSelector) {
     this._nameElement = document.querySelector(nameSelector);
     this._jobElement = document.querySelector(jobSelector);
+    this._avatarElement = document.querySelector(avatarSelector);
   }
   
   getUserInfo() {
     const userInfo = {
       name: this._nameElement.textContent,
       job: this._jobElement.textContent,
+      avatar: this._avatarElement.textContent,
     };
     return userInfo;
   }
 
-  setUserInfo(name, job) {
+  setUserInfo(name, job, avatar) {
     this._nameElement.textContent = name;
     this._jobElement.textContent = job;
+    this._avatarElement.textContent = avatar;

```

Comment:
`    this._avatarElement.textContent = avatar;
`

This is not needed here because the avatar doesn't have `textContent`